data-explorer/ingest-json-formats.md

```diff
@@ -23,7 +23,7 @@ Azure Data Explorer supports two JSON file formats:
 * `multijson`: Multi-lined JSON. The parser ignores the line separators and reads a record from the previous position to the end of a valid JSON.
 
 > [!NOTE]
-> When ingesting using the [ingestion wizard](ingest-data-wizard.md), select **Ignore data format errors**, to ingest data in JSON format. If you leave this check box unselected, the data will be ingested in multijson format. For more information, see [edit the schema in the ingestion wizard](ingestion-wizard-existing-table.md#edit-the-schema).
+> When ingesting using the [ingestion wizard](ingest-data-wizard.md), select **Ignore data format errors**, to ingest data in JSON format. This selection ignores ill-formatted rows errors by ignoring the incorrect line, and fully ingesting the rest of the file. If you leave this check box unselected, the data will be ingested in multijson format. For more information, see [edit the schema in the ingestion wizard](ingestion-wizard-existing-table.md#edit-the-schema).
```

Comment:
```suggestion
> When ingesting using the [ingestion wizard](ingest-data-wizard.md), select **Ignore data format errors**, to ignore rows that are incorrectly formatted, but fully ingest the rest of the data in JSON format. Otherwise, the data is ingested in MultiJSON format. For more information, see [edit the schema in the ingestion wizard](ingestion-wizard-existing-table.md#edit-the-schema).
```

Reply:
I would probably phrase this as:

When ingesting using the [ingestion wizard](ingest-data-wizard.md), the default format flavor is `multijson`.
This format allows multiline JSON records, as well as arrays of JSON records. When parsing error is encountered, the entire file is discarded.
If you are using JSON-line format (JSON records are single line, and every line is a well-formed JSON record), and would like to accommodate for an occasional malformed record, select **Ignore data format errors** to skip the malformed records while ingesting the valid ones.